.eslintrc.js

```diff
@@ -1,37 +1,24 @@
-module.exports =  {
-  parser:  '@typescript-eslint/parser',  // Specifies the ESLint parser
-  extends:  [
-    'plugin:@typescript-eslint/recommended',  // Uses the recommended rules from the @typescript-eslint/eslint-plugin
-    'prettier/@typescript-eslint',  // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
-    'plugin:prettier/recommended',  // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
-    'airbnb', 'prettier'
-  ],
-  parserOptions:  {
-    ecmaVersion:  2018,  // Allows for the parsing of modern ECMAScript features
-    sourceType:  'module',  // Allows for the use of imports
-  },
-  rules:  {
-    "max-len": ["error", {"code": 120}],
-    "react/jsx-indent": ["off"],
-    "react/jsx-filename-extension": ["off"],
-    "react/prop-types": ["off"],
-    "import/no-extraneous-dependencies": ["error", {
-      "devDependencies": [
-        ".storybook/**",
-        "stories/**",
-      ]
-    }],
-  },
-  settings:  {
-    "import/parsers": {
-      "@typescript-eslint/parser": [".ts", ".tsx"],
+module.exports = {
+    env: {
+        browser: true,
+        es6: true,
     },
-    "import/resolver": {
-      "typescript": {},
+    extends: 'eslint:recommended',
+    globals: {
+        Atomics: 'readonly',
+        SharedArrayBuffer: 'readonly',
     },
-    react:  {
-      version:  'detect',  // Tells eslint-plugin-react to automatically detect the version of React to use
+    parser: 'babel-eslint',
+    parserOptions: {
+        sourceType: 'module',
+        ecmaFeatures: {
+            jsx: true,
+            tsx: true,
+        },
+        ecmaVersion: 2018,
+    },
+    plugins: ['react'],
+    rules: {
+        'no-unused-vars': 'off', // TODO remove
```

Comment:
pls create a ticket on Asana for removing this rule once you will find why it didn't work well in first place

Reply:
done